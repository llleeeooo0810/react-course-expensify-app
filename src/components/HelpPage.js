import React from 'react';

const HelpPage = () => (
    <div>
        <p>This is from help page</p>
    </div>
);

export default HelpPage;