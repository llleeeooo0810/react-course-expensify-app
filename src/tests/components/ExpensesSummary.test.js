import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should correctly ExpensesSummary with one expense', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={12990} />);
    expect(wrapper).toMatchSnapshot();
});

test('should correctly ExpensesSummary with multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={15} expensesTotal={512973217990} />);
    expect(wrapper).toMatchSnapshot();
});