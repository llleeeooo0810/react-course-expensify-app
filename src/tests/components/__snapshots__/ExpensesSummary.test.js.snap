// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly ExpensesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    15
     
    expenses
     totalling 
    $5,129,732,179.90
  </h1>
</div>
`;

exports[`should correctly ExpensesSummary with one expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $129.90
  </h1>
</div>
`;
